nix/helix-core-api.nix

```diff
@@ -4,7 +4,7 @@
     aarch64-darwin = fetchzip {
       name = "helix-core-api";
       url = "https://filehost.perforce.com/perforce/r23.1/bin.macosx12arm64/p4api-openssl1.1.1.tgz";
-      hash = "sha256-9jyecw8S98nKRy3mehrMlndaggu3suEyQIZq8pz+dX0=";
+      hash = "sha256-0jU51a239Ul5hicoCYhlzc6CmFXXWqlHEv2CsTYarS0=";
```

Comment:
Updating these because the nix build pipeline fails 🤷‍♂️  seems like Perforce updated these?

Reply:
Got a fix here for all 3 platforms https://github.com/sourcegraph/sourcegraph/pull/63805/files